plone/namedfile/browser.py

```diff
@@ -43,7 +51,12 @@ def publishTraverse(self, request, name):
     def __call__(self):
         file = self._getFile()
         self.set_headers(file)
-        return stream_data(file)
+        if not INamedBlobFile.providedBy(file) or not HAS_PAB:
```

Comment:
Where comes this INamedBlobFile.providedBy from in this context, could this ever become False?